web/cypress/e2e/auto-tax-filings.spec.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+import {
+  clickModalSaveButton,
+  completeBusinessStructureTask,
+  openFormationDateModal,
+  selectDate,
+  selectLocation,
+} from "@businessnjgovnavigator/cypress/support/helpers/helpers";
+import { completeNewBusinessOnboarding } from "@businessnjgovnavigator/cypress/support/helpers/helpers-onboarding";
+import { LookupIndustryById } from "@businessnjgovnavigator/shared/";
+import { onDashboardPage } from "cypress/support/page_objects/dashboardPage";
+
+describe("auto tax filing [feature] [all] [group4]", () => {
+  beforeEach(() => {
+    cy.loginByCognitoApi();
+  });
+
+  it("successfully completes auto tax filing with business name and tax Id", () => {
+    const industry = LookupIndustryById("cosmetology");
+    const legalStructureId = "limited-liability-company";
+    const businessName = "Make Me Pretty";
+
+    completeNewBusinessOnboarding({ industry });
+
+    completeBusinessStructureTask({ legalStructureId });
+
+    onDashboardPage.getEditProfileLink().should("exist");
+    cy.get('[data-task="apply-for-shop-license"]').first().click();
+    cy.get('button[data-testid="cta-secondary"]').first().click();
```

Comment:
Why are we interacting with a license task? It doesn't look like this is doing anything to modify the data.